apps/dashboard/src/lib/components/QuestionContainer/index.svelte

```diff
@@ -8,8 +8,9 @@
   export let onClose = () => {};
   export let scrollToQuestion = false;
   export let points = undefined;
+  export let explanation = undefined;
   export let hasError = false;
-  export let onPointsChange = () => {};
+  export let onChange = () => {};
```

Comment:
What did you rename this?

Reply:
Oh this was a mistake, in summary i initially used a different approach for that so i changed the prop, i've pushed the fix for it now